lib/ansible/utils/path.py

```diff
@@ -18,6 +18,7 @@
 __metaclass__ = type
 
 import os
+import re
```

Comment:
I think we don't need the re import anymore as we're no longer defining a regex here.